services/CurrentTime/include/ble-service-current-time/CurrentTimeService.h

```diff
@@ -43,6 +44,13 @@
  */
 class CurrentTimeService {
 public:
+    mbed::Timer timer;
```

Comment:
This is of no interest for the user and we don't them to mess with the timer. This should be a private member.